oeps/architectural-decisions/oep-0064-mobile-application-codebase-modernization.rst

```diff
@@ -0,0 +1,103 @@
+
+
+Specification 
+************* 
+
+The existing mobile repositories are maintained by edX / 2U and can be accessed here:
+
+* https://github.com/openedx/edx-app-ios
+* https://github.com/openedx/edx-app-android
+
+* Support for the existing mobile applications will continue through the Redwood release. Starting with the release after Palm, we will designate the new repositories as the default Open edX mobile applications to encourage adoption and use of the new applications for community team members.
+* Once available, we will link to documentation within the new repositories to describe transition options for community members migrating from existing builds of the older codebases.
+* After the publication of release after Palm we will update the original repositories with the suffix ‘archived’ to help make clear that these are no longer the default mobile application builds moving forward. A corresponding DEPR ticket linked to this OEP once it exists. 
+
+The new mobile repositories are maintained by Raccoon Gang and can be accessed here:
+* https://github.com/raccoongang/educationx-app-ios
+* https://github.com/raccoongang/educationx-app-android
+
+* The new repositories will be migrated to the openedx github organization under the name openedx-app-ios and openedx-app-android.
+* The new repositories will be maintained by the Racoon Gang team with Marco Morales serving as the product lead for the project.
+* Relevant funded contributions and projects relation to the new mobile repositories will be linked from their main project readme as a way to help community members find ongoing and future efforts
+
+
+Rationale
+*********
+
+Expanded access and opportunities for learners around the globe means meeting learners where they are, and for many people that means on their only digital device, their phone. Furthermore, enabling learners to continue to learn on their desktop or laptop computers in addition to a full featured mobile experience allows us to support full day learning workflows for those with multiple devices.
+
+Similar to the way the edx-platform has undergone a modernization process over the past few years with updated build processes, micro services, micro frontends, design system tools and more, the mobile experiences will benefit from a modern architecture, being written in the latest development language tools, and set up for increased pluggability and contribution as well.
+
+Backward Compatibility
+**********************
+The existing mobile applications will continue to be supported by edX / 2U for the foreseeable future. We do not yet have a clear timeline for when edX / 2U might switch over to the new applications, but their focus is on rollout and delivery of their latest improvements to the existing experience including the commerce capabilities that have been added recently. 
```

Comment:
We dont actually know who else in the community is using the existing apps, this note was meant to convey continued support for these applications until edx / 2U decides to move to the new applications

I can strike if this concern doesn't feel core to what the OEP should be covering though!

Reply:
I think I understand the point, but as-written it feels too "point in time" - OEPs should be lasting documents. At minimum I would strike the point about commerce capabilities because I don't think edX/2U is planning to support that for the community.

Reply:
makes sense! updating. thanks for feedback on this.